package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "setup",
+  "version": "1.0.0",
+  "description": "Setting up GraphQL with express and GraphiQL",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/feature/setup-graphql.git"
+  },
+  "keywords": [
+    "graphql",
+    "express"
+  ],
+  "author": "Sam",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/feature/setup-graphql/issues"
+  },
+  "homepage": "https://github.com/feature/setup-graphql#readme",
+  "dependencies": {
+    "express": "^4.21.0",
```

geralmente fixamos a versão das dependências para evitar que o código quebre sozinho com mudanças nas libs
```suggestion
    "express": "4.21.0",
```